catalog.html

```diff
@@ -15,96 +15,111 @@
 </head>
 
 <body class="page-body">
-  <header class="header">
-    <nav class="navigation">
-      <a href="index.html">
-        <img src="img/logo.svg" alt="Логотип главной страницы" width="138" height="70">
-      </a>
-      <ul class="items main-navigation">
-        <li><a href="#">Информация</a></li>
-        <li><a href="#">Фото и видео</a></li>
-        <li><a href="#">Карта штата</a></li>
-        <li><a href="#" class="current hotels">Гостиницы</a></li>
-      </ul>
-    </nav>
-  </header>
+  <div class="container">
+    <header class="header">
+      <nav class="navigation">
+        <a href="index.html">
+          <img src="img/logo.svg" alt="Логотип главной страницы" width="138" height="70">
+        </a>
+        <ul class="items main-navigation">
+          <li><a href="#">Информация</a></li>
+          <li><a href="#">Фото и видео</a></li>
+          <li><a href="#">Карта штата</a></li>
+          <li><a href="#" class="current hotels">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
 
-  <main>
-    <section>
-      <h3 class="title">Найдено:3</h3>
-      <p class="sort-item">Сортировка:</p>
-      <ul class="items list-sort">
-        <li><a href="#" class="active sort-item">По цене</a></li>
-        <li><a href="#" class="sort-item">По типу</a></li>
-        <li><a href="#" class="sort-item">По рейтингу</a></li>
-      </ul>
+    <main>
+      <section>
+        <div class="container-sorts">
+          <h3 class="title">Найдено:3</h3>
+          <p class="sort-item">Сортировка:</p>
+        <ul class="items list-sort">
+          <li class="inline"><a href="#" class="active sort-item">По цене</a></li>
```

Comment:
плохой пример названия класса, sort-item будет лучше
```suggestion
          <li class="sort-item"><a href="#" class="active sort-link">По цене</a></li>
```